docs/conf.py

```diff
@@ -65,7 +65,11 @@
                         'kaolin.cuda.three_nn',
                         'kaolin.cuda.tri_distance',
                         'kaolin.cuda.mesh_intersection',
-                        'kaolin.graphics.nmr.cuda.rasterize_cuda']
+                        'kaolin.graphics.nmr.cuda.rasterize_cuda',
+                        'kaolin.graphics.softras.soft_rasterize_cuda',
+                        'kaolin.graphics.dib_renderer.cuda.rasterizer',
+                        'graphics.nmr.cuda.rasterize_cuda',
```

Comment:
I'm not sure to understand, does `graphics` exists out of `kaolin/` ?

Reply:
Good catch! That was there for debugging while I fixed a bad module description.